README.md

```diff
@@ -28,85 +30,90 @@ The following table contains nrich versions with corresponding minimum Java and
 | 1.3.x         | 1.8          | 2.6.6               |
 | 1.4.x         | 1.8          | 2.6.6               |
 
-
 ## Modules
 
-nrich is composed of following modules:
+Nrich is composed of following modules:
 
-### nrich-bom
+### [nrich-bom](nrich-bom/README.md)
 
 Provides managed dependencies for all nrich modules as well as for libraries used inside those modules that are not covered by Spring Boot dependencies.
 
-### nrich-core-api
+### [nrich-core-api](nrich-core-api/README.md)
 
-Contains common classes that are used throughout library
+It contains common classes that are used throughout the library.
```

Comment:
I'm not sure about using `it` in this context. It sounds a bit strange to me. Maybe think about something else?
There are multiple places with this so please check them if we agree to make the change.

Reply:
I agree and i commited changes to every `it` that I found incorrect. Instead I referenced the module to which the `it` was referencing to or replaced `it` with other formation.